tests/Admin/Admin_Page_Tests.php

```diff
@@ -50,6 +49,11 @@ public function test_add_page() {
 		$this->assertEquals( 'tools.php', $parent_pages['plugin-check'] );
```

Comment:
See above, by using the `$hook` as return value, we can also assert here that the hook to initialize the page was added.